src/renderer/services/chain/fees/swap.ts

```diff
@@ -22,18 +21,21 @@ import { getChainAsset } from '../../../helpers/chainHelper'
 import { LiveData, liveData } from '../../../helpers/rx/liveData'
 import * as BNB from '../../binance'
 import * as BTC from '../../bitcoin'
+import * as ETH from '../../ethereum'
 import * as LTC from '../../litecoin'
 import * as THOR from '../../thorchain'
 import { FeesLD, Memo, SwapFeesLD } from '../types'
 
 const reloadSwapFees = () => {
   BNB.reloadFees()
   BTC.reloadFees()
+  ETH.reloadFees()
   THOR.reloadFees()
   LTC.reloadFees()
```

Comment:
^ `LTC.reloadFees()` -> `LTC.reloadfeesWithRates(memo)`

It should be fixed w/o PR.